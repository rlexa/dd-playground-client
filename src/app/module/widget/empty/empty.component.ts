import {ChangeDetectionStrategy, Component} from '@angular/core';

@Component({
  selector: 'app-empty',
  template: '<div>...</div>',
  changeDetection: ChangeDetectionStrategy.OnPush,
  standalone: true,
})
export class EmptyComponent {}
