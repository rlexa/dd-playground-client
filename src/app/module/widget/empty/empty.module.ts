import {NgModule} from '@angular/core';
import {EmptyComponent} from './empty.component';

@NgModule({imports: [], exports: [EmptyComponent], declarations: [EmptyComponent]})
class EmptyModule {}

export {EmptyModule, EmptyComponent};
