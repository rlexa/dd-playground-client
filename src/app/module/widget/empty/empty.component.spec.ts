import {ComponentFixture} from '@angular/core/testing';
import {MockBuilder, MockRender} from 'ng-mocks';
import {EmptyComponent} from './empty.component';

describe('BuildComponent', () => {
  let fixture: ComponentFixture<EmptyComponent>;

  beforeEach(() => MockBuilder(EmptyComponent));

  beforeEach(() => {
    fixture = MockRender(EmptyComponent);
    fixture.detectChanges();
  });

  test('renders', () => expect(fixture).toMatchSnapshot());
});
