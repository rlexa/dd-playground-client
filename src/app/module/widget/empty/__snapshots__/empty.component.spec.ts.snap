// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BuildComponent renders 1`] = `
<mock-render>
  <app-empty>
    <div>
      ...
    </div>
  </app-empty>
</mock-render>
`;
